library/build.gradle

```diff
@@ -15,6 +15,8 @@ android {
 dependencies {
   api "com.android.support:recyclerview-v7:$supportlib_version"
   implementation "com.android.support:appcompat-v7:$supportlib_version"
+  implementation "android.arch.lifecycle:extensions:$architecture_version"
```

Comment:
These 2 deps seem to be unnecessary.

Reply:
ah never mind, you will use these deps in the future

Reply:
Oops, looks like I forgot to put those in the next PR. Sorry! 😊